pandas/tests/series/test_dtypes.py

```diff
@@ -518,3 +518,9 @@ def test_values_compatibility(self, data):
         result = pd.Series(data).values
         expected = np.array(data.astype(object))
         tm.assert_numpy_array_equal(result, expected)
+
+    def test_uint_drop(self):
+        for type in ["uint8", "uint16", "uint32", "uint64"]:
```

Comment:
Could you:

  1. Parameterize this instead of doing a loop within the function body
  2. Change `type` to `type_` so as not to shadow the builtin
  3. Add the issue number as a comment?

Reply:
you can pop in ``uint_dtype`` to the argument and it will just work